Cargo.toml

```diff
@@ -10,32 +10,33 @@ license = "MIT"
 # Remember to also update in appveyor.yml
 version = "0.7.1"
 # Remember to also update in checksums.md
-authors = ["nabijaczleweli <nabijaczleweli@gmail.com>", "Zachary Dremann <dremann@gmail.com>", "Chris Moore", "Daniel Alley <dalley@redhat.com>"]
+authors = ["nabijaczleweli <nabijaczleweli@gmail.com>", "Zachary Dremann <dremann@gmail.com>", "Chris Moore", "Daniel Alley <dalley@redhat.com>, Paul Bragin <zeusmods@protonmail.com>"]
 exclude = ["*.enc"]
 
 
 [dependencies]
-futures-cpupool = "0.1"
-blake2-rfc = "0.2"
-once_cell = "1.5"
-tabwriter = "1.2"
-num_cpus = "1.13"
-futures = "0.1"
-walkdir = "2.3"
-crc32c = "0.5"
+futures-cpupool = "0.1.8"
```

Comment:
sort these reverse-length and drop the last version component

Reply:
Eh... Can you explain a bit more? I don't get it

Reply:
These are sorted length-decreasing then alphabetical: keep that ordering. And remove the patch version component from the specs.